lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -329,6 +350,16 @@ def __init__(self, *args, zs=(), zdir='z', **kwargs):
         self.set_3d_properties(zs, zdir)
 
     def set_3d_properties(self, verts, zs=0, zdir='z'):
+        """
+        Set the *z* position and direction of the patch.
+
+        Parameters
+        ----------
+        z : float
```

Comment:
```suggestion
        zs : float
```
Also, it's allowed to be an array that matches `verts`, I think?